### ISSUE TITLE
[ADE Support]: OCTO VA: Investigate use of AI - Figma Sites Notification

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#109557]
> [!NOTE]
> **QUICK SUMMARY:** A notification needs to be communicated about the lack of accessibility with Figma Sites to the larger VA Design community. 

## Description of the Work
- :warning: Create a PSA re: Figma Sites and Accessibility
- @coforma-jamie to help draft and @artsymartha68 to review and help distribute. 

### Background
- **Figma Sites was just announced and it is not accessible at this time:** 
    - **Caution**: Do not be tempted to use for prototyping.
        - The code is not there yet and is bad for accessibility.
        - It lacks proper semantic HTML.
        - They are not yet accessible.

## About the Team 
> [!TIP]
> _Delete About the Team if internal and not needed._

- **Team name**:  OCTO
- **Product name:** Figma Sites
- **Slack Channels:** [#accessibility-help](https://dsva.slack.com/archives/C8E985R32) channel and [#design ](https://dsva.slack.com/archives/C0NGDDXME)channel 

## Supporting Artifacts 
- [Adrian Roselli Blog](https://adrianroselli.com/2025/05/do-not-publish-your-designs-on-the-web-with-figma-sites.htm)
- [UX Design Article](https://uxdesign.cc/figma-sites-when-accessibility-is-an-afterthought-070ba3b41553)
- [Slack thread aggregation](https://dsva.slack.com/archives/C08B367JLLT/p1747054462328199)

## ADE Tasks
> [!TIP]
> Customize ADE Tasks as needed. 
- [x] Collaborate `task`
- [ ] ADE Support `task`
- [ ] Document `task`
- [ ] Review `task`
- [ ] Revisit `task` to see if it is made to be accessible in the future, with new ticket in `backlog`. 

## Acceptance Criteria
> [!TIP]
> Customize ADE Acceptance Criteria as needed. 
- [x] Notify Martha there is an issue with Figma Sites. 5/12/2025
- [x] Notify team internally ([thread](https://dsva.slack.com/archives/C08B367JLLT/p1747054462328199) in ADE Slack Internal Channel) 5/12/2025 - @sara-amanda 
- [ ] Notify ADE Companies: Coforma [ ], Oddball [x], Pluribus [ ] and AdHoc [ ] 
- [ ] @coforma-jamie to draft wording for the announcement/warning. 
- [ ] @artsymartha68 to review and distribute. 

### Ticket Updates
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [x] **Read** Important Notifications (ADE Internal) at the bottom of this ticket.
- [x] **Delete** Important Notifications (ADE Internal) at the bottom of this ticket, after reading.

**Comment by coforma-jamie** [comment]
@artsymartha68 please zhuzh as needed:

> Last week, Figma released a new product, Figma Sites, that allows Figma users to turn mocks into web sites within the Figma platform.
> 
> Unfortunately, the websites it generates **are not accessible.** Accessibility specialists have uncovered many issues, including:
> - No semantic markup (all objects are `<divs>`, including headings)
> - Issues with reading order
> - Links open in new tabs by default, with no way to change the setting or warn the user
> 
> Because of these critical accessibility issues, we ask that you **don't use Figma sites for testing with users of assistive technology.**
> 
> We'll keep track of Figma Sites' evolution and if the (many) critical issues are corrected, we will let you know.
> 
> References:
> - [Emily Lawes' writeup on LinkedIn](https://www.linkedin.com/posts/emilylawes_accessibility-figma-a11y-activity-7326207820545028096-0RUN/?utm_source=productpicnic.beehiiv.com&utm_medium=referral&utm_campaign=figma-dreamweaver)
> - [Do Not Publish Your Designs on the Web with Figma Sites](https://adrianroselli.com/2025/05/do-not-publish-your-designs-on-the-web-with-figma-sites.html)
> - [Figma Dreamweaver](https://productpicnic.beehiiv.com/p/figma-dreamweaver)
